src/otx/algo/detection/heads/custom_atss_head.py

```diff
@@ -8,39 +8,98 @@
 from typing import TYPE_CHECKING
 
 import torch
-from mmdet.models.dense_heads.atss_head import ATSSHead
-from mmdet.models.utils.misc import multi_apply
-from mmdet.registry import MODELS
-from mmdet.structures.bbox.bbox_overlaps import bbox_overlaps
-from mmdet.utils.dist_utils import reduce_mean
-from torch import Tensor
+from mmcv.cnn import ConvModule
```

Comment:
Don't we use `from otx.algo.modules.conv_module import ConvModule`?

Reply:
I'll do that when full decoupling